Carwash/CW_places/urls.py

```diff
@@ -6,6 +6,7 @@
 router = routers.DefaultRouter()
 router.register(r'cw_places', CWPlacesView)
 
+"""Define the mapping between URLs and views. """
 urlpatterns = [
-    path('', include(router.urls)),
+    path('api/', include(router.urls)),
```

Comment:
what is the URL we receive for  GET list of CW places?

Reply:
This is the URL for GET - /api/cw_places/